packages/wasm/crate/src/planner.rs

```diff
@@ -91,7 +91,7 @@ impl ActionList {
     }
 
     fn fee_estimate(&self, gas_prices: &GasPrices, fee_tier: &FeeTier) -> Fee {
-        let base_fee = Fee::from_staking_token_amount(gas_prices.fee(&self.gas_estimate()));
+        let base_fee = gas_prices.fee(&self.gas_estimate());
```

Comment:
`GasPrices::fee()` now returns a `Fee` instead of an `Amount`. I figured `Fee::from_staking_token_amount()` isn't needed, since the fee from a `GasPrices` should always be in the staking token.

Reply:
planner logic will need to change again, so that's alright